src/compile_torture_tests.py

```diff
@@ -21,24 +21,6 @@
 
 import testing
 
-# TODO: Pass this in from build.py
-INSTALL_DIR = os.path.join(os.getcwd(),
```

Comment:
Wouldn't a simpler fix be to simply replace this line with:

```
SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
INSTALL_DIR = os.path.join(SCRIPT_DIR,  'src', 'work', 'wasm-install', 'sysroot')
```

(i.e. its the bug just that getcwd() was using rather than SCRIPT_DIR?)